libs/src/oracle2/services/index.ts

```diff
@@ -1,5 +1,4 @@
-export * as oracleV1Gql from "./oracleV1/gql";
-export * as oracleV3Gql from "./oracleV3/gql";
-export * as oracleSkinnyGql from "./oracleSkinny/gql";
```

Comment:
the interfaces for skinny and oo-v1 are actually identical. Pablo has a pr coming that rectifies this in the graph schema. for now we can use our mocks to see this working as expected